packages/@uppy/companion/src/server/controllers/send-token.js

```diff
@@ -1,4 +1,6 @@
 const { URL } = require('url')
+const serialize = require('serialize-javascript')
+
 const tokenService = require('../helpers/jwt')
 const { hasMatch, sanitizeHtml } = require('../helpers/utils')
```

Comment:
Is `sanitizeHtml` still used?

Reply:
right! good catch. I'll remove the whole function. eslint would have caught unused sanitizeHtml but we only use warning for most rules 😄 we need to eventually go through all warnings and fix them so we can upgrade more eslint rules to error. now it's just too many warning so I tend to ignore them all

Reply:
https://github.com/transloadit/uppy/pull/3095 ;)

Reply:
Awesome!!